sphinx_gallery/gen_gallery.py

```diff
@@ -68,6 +68,7 @@
     'junit': '',
     'log_level': {'backreference_missing': 'warning'},
     'inspect_global_variables': True,
+    'ignore_repr_classes': r'',
```

Comment:
Would `ignore_repr_types` be clearer since we check `type(...)` ?

Reply:
I am confused between [class and type](https://www.python.org/download/releases/2.2/descrintro/) now. Should I then change the documentation to use the term 'type' as well then?

Reply:
Yeah I'm not sure either. But I think in this case "type" is safer/clearer because we don't traverse all subclasses, we just check `type(x)`. One might assume if we named it `ignore_repr_classes` and they had `ignore_repr_classes='object'`, then *no* `__repr__`s would ever be printed. However, based on our implementation only in the (weird) case that someone returned `object()` nothing would get printed.

Reply:
... In other words, asking "is this thing class X?", which is implied by `classes` in the current naming scheme, is really like doing `isinstance(thing, X)` rather than `type(thing) == X`. We currently check the latter, which I think is desired behavior, but makes `_classes` a misnomer.

Reply:
Ah I see, thanks. Changing now